.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
```

Comment:
Why are we using CircleCI to test a GitHub action?

Reply:
No other reason then it's just how we have everything setup. If we feel included we can change it in another pr.